finder-app/finder.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+: '
+  *****************************************************************************/
+​ * Copyright​ ​(C)​ ​2022​ ​by​ ​Nileshkartik Ashokkumar
+​ *
+​ * ​​Redistribution,​ ​modification​ ​or​ ​use​ ​of​ ​this​ ​software​ ​in​ ​source​ ​or​ ​binary
+​ * ​​forms​ ​is​ ​permitted​ ​as​ ​long​ ​as​ ​the​ ​files​ ​maintain​ ​this​ ​copyright.​ ​Users​ ​are
+​ ​* ​permitted​ ​to​ ​modify​ ​this​ ​and​ ​use​ ​it​ ​to​ ​learn​ ​about​ ​the​ ​field​ ​of​ ​embedded
+​ * software.​ Nileshkartik Ashokkumar ​and​ ​the​ ​University​ ​of​ ​Colorado​ ​are​ ​not​ ​liable​ ​for
+​ ​* ​any​ ​misuse​ ​of​ ​this​ ​material.
+​ * 
+ *****************************************************************************/
+​ ​*​ ​@file​ ​finder.sh
+​ ​*​ ​@brief​ ​a shell script that takes 2 arguments from the user, 1 st argument is file path and 2nd argument is a word
+ * The Functionality of this shell script is to count the presence of the 2nd argument i.e.  word in the files mentioned by the user in the 1st argument
```

Comment:
@Nileshkartik95 the description here is good, returning a description like this in the case where arguments are invalid might be more helpful for the user.